src/main.js

```diff
@@ -6,8 +6,12 @@ import store from "./store";
 import upperFirst from "lodash/upperFirst";
 import camelCase from "lodash/camelCase";
 
+import $ from "jquery";
+window.$ = window.jQuery = $;
+
```

Comment:
Bitte kein jquery in diesem Projekt nutzen.

Reply:
Das Paket tauch in `package.json` nur auf, weil das "legacy SmartHome" diese benötigt.